Error in build with node typing

Running build command I get this error message:

`node_modules/@google-cloud/firestore/types/firestore.d.ts:1367:15 - error TS2503: Cannot find namespace 'NodeJS'.`

and when I install @google-cloud/firestore I get this message:

`error TS2688: Cannot find type definition file for 'node'.`

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
   "author": "Jacek B. Budzynski (jacek.b.budzynski@gmail.com)",
   "license": "BSD-2-Clause",
   "devDependencies": {
-    "@google-cloud/firestore": "^4.7.1"
+    "@google-cloud/firestore": "^4.7.1",
+    "@types/node": "^14.14.7"
   }
 }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -7,7 +7,8 @@
     "sourceMap": true,
     "strict": true,
     "declaration": true,
-    "target": "es2017"
+    "target": "es2017",
+    "types": ["node"]
   },
   "compileOnSave": true,
   "include": [

